src/config.ml

```diff
@@ -473,6 +474,17 @@ module Conf_map = struct
        ensure_not Tls_crypt_v2_server
          "server tls-crypt-v2 key passed in tls-crypt-v2")
 
+  let pp_param ppf s =
+    (* XXX(reynir): a_param backslash escaping is probably not correct *)
+    let escape ppf s =
+      Fmt.char ppf '"';
+      Fmt.(list ~sep:(any "\\\"") string) ppf (String.split_on_char '"' s);
```

Comment:
I find it more readable to use the other string syntax:

```suggestion
      Fmt.(list ~sep:(any {|\"|}) string) ppf (String.split_on_char '"' s);
```

Reply:
Addressed in f2b331bfd68139337f4060f87503c2bf3c900ace